console/v1/register.go

```diff
@@ -32,6 +32,7 @@ func Resource(resource string) schema.GroupResource {
 func addKnownTypes(scheme *runtime.Scheme) error {
 	scheme.AddKnownTypes(GroupVersion,
 		&ConsoleLink{},
+		&ConsoleLogLink{},
```

Comment:
I mentioned on slack, not sure I'm happy with `ConsoleLink` and `ConsoleLogLink`.  The names are very similar, and the word `Log` doesn't really clarify intent.

The new one is something more like `ConsoleExternalServiceLink`. (`Service` may also be an overloaded term, however).  `ConsoleLink` is really for simple menu links.  The new link is for much more contextual link generation that could be for logs, monitoring, probably other kinds of services.

Reply:
This is so specific to logs, I'm not sure "External Service Link" is good. Do we have any other specific kinds of external services we're planning to add? Without knowing what will come later, it's difficult to say whether the properties here will work for other kinds of services. It's going to be hard to add a different kind of external service later if we need completely different properties.

I agree the names are too close. Maybe `ConsoleExternalLogLink` is better?

Reply:
From `resourceName`, `containerName`, `resourceNamespace`, `podLabels` it seems like this is pretty generic.  If we had another use case arise, would we expand this further, or would we rather create a third CRD?  I'd lean towards just expanding available parameters here. Otherwise, we would end up with `ConsoleLink`, `ConsoleLogLink`, `ConsoleFooLink`, seems like potential for more confusion.

Put another way, do we want to make it clear that we do not desire to add any other resources, but would rather expand this one (if future additional needs arise)?